stories/DateRangePicker.js

```diff
@@ -84,8 +84,13 @@ storiesOf('DateRangePicker (DRP)', module)
   })
   .addWithInfo('non-english locale (Persian)', () => {
     moment.locale('fa');
+    momentJalaali.loadPersian({ dialect: 'persian-modern', usePersianDigits: true });
```

Comment:
Can you confirm whether this impacts moment usage at all?

Reply:
It just impact on Persian example.

Reply:
What i mean is, does this call affect moment, or only momentJalaai?

Reply:
No, just momentJalaai.